src/jit/codegencommon.cpp

```diff
@@ -4246,7 +4246,9 @@ void            CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg,
 #if defined(_TARGET_ARM_)
             insCopy = INS_vmov;
 #elif defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
-            insCopy = INS_mov;
+            insCopy = INS_movdqa;
+
+
 #else
 #error Error. Wrong architecture.
 #endif
```

Comment:
It was a way to avoid the #error ifdef. Changed it not to have the unnecessary code.